wolfcrypt/src/asn.c

```diff
@@ -864,7 +868,7 @@ WOLFSSL_LOCAL int GetMyVersion(const byte* input, word32* inOutIdx,
 {
     word32 idx = *inOutIdx;
 
-    WOLFSSL_ENTER("GetMyVersion");
+    //WOLFSSL_ENTER("GetMyVersion");
```

Comment:
Removed. Also extremely version for debugging / benchmark. I personally don't see the value of logging this.